src/components/ChangePlan/Checkout/PaymentMethod/PaymentMethod.js

```diff
@@ -40,6 +40,25 @@ const paymentMethods = [
   },
 ];
 
+export const getCreditCardBrand = (creditCardNumber) => {
```

Comment:
```
export const getCreditCardBrand = (creditCardNumber) => {
  // first check for MasterCard
  if (/^5[1-5]/.test(creditCardNumber)) {
    return  'mastercard';
  }
  
  // then check for Visa
  if (/^4/.test(creditCardNumber)) {
    return 'visa';
  }

  // then check for AmEx
  if (/^3[47]/.test(creditCardNumber)) {
    return 'amex';
  }
  
  return 'unknown';
};
```

What do you think about this, @cbernat @nreal ?

Reply:
It seems easier to me, besides that I usually try to avoid using the else, because it reduces the cyclomatic complexity of the function

Reply:
you can try this function for credit card types and for 'unknown' case.

Reply:
In my opinion I agree, is more easy to read the algorithm, thanks

Reply:
done